pages/common/xml-format.md

```diff
@@ -0,0 +1,28 @@
+# xml-format
+
+> Format an XML document.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Format an XML document, indenting with tabs:
+
+`xml format --indent-tab {{path/to/input.xml|URI}} >{{path/to/output.xml}}`
+
+- Format an HTML document, indenting with 4 spaces:
+
+`xml format --html --indent-spaces {{4}} {{path/to/input.html|URI}} >{{path/to/output.html}}`
+
+- Recover parsable parts of a malformed XML document, without indenting:
+
+`xml format --recover --noindent {{path/to/malformed.xml|URI}} >{{path/to/recovered.xml}}`
+
+- Format an XML document from standard input, removing the DOCTYPE declaration:
```

Comment:
Perhaps this will read better?
```suggestion
- Format an XML document from the standard input, removing the DOCTYPE declaration:
```

Reply:
- Format an XML document from the standard input, removing the DOCTYPE declaration:

Reply:
xml.md:- Execute a subcommand with input from a file or URI, printing to stdout:
xml.md:- Execute a subcommand using stdin and stdout:
xml-depyx.md:- Convert a PYX document from stdin to XML format:
xml-escape.md:- Escape special XML characters from stdin:
xml-format.md:- Format an XML document from stdin, removing the DOCTYPE declaration:
xml-pyx.md:- Convert an XML document from stdin to PYX format:
xml-unescape.md:- Unescape special XML characters from stdin: